File: pinyougou_parent/pinyougou_sellergoods_interface/src/main/java/com/pinyougou/sellergoods/service/OrderService.java
package com.pinyougou.sellergoods.service;
import java.util.List;
import com.pinyougou.pojo.TbOrder;							  
import entity.PageResult;
import entity.Result;
/**
 * order服务层接口
 * @author jieweili
 *
 */
public interface OrderService {

    /**
     * 返回全部列表(已弃置)
     * @return
     */
    List<TbOrder> findAll();
 
    /**
     * 返回分页对象(过时)
     * @param pageNum
     * @param pageSize
     * @return
     */
	 
    PageResult findPage(Integer pageNum, Integer pageSize);

    /**
     * 添加数据
     * @param order
     * @return
     */
    public Result saveOrder(TbOrder order);

    /**
     * 根据id查询一个对象
     * @param id
     * @return
     */
    TbOrder getOrderById(Long id);

    /**
     * 修改对象
     * @param order
     * @return
     */
    Result updateOrder(TbOrder order);

    /**
     * 批量删除
     * @param idList
     * @return
     */
    Result removeOrder(Long[] idList);

    /**
     * 根据条件分页查询
     * @param order
     * @param pageNum
     * @param pageSize
     * @return
     */
    PageResult listOrderByExample(TbOrder order, Integer pageNum, Integer pageSize);
 
}

File: pinyougou_parent/pinyougou_manager_web/src/main/webapp/js/service/orderService.js
app.service('orderService', function ($http) {
    //旧的查询方法(弃置)								   
    this.findPage = function (pageNum, pageSize) {
        return $http.get('../order/findPage.do?pageNum=' + pageNum + '&pageSize=' + pageSize)
    };

    //查询方法,读取列表数据绑定到表单中
    this.search = function (pageNum, pageSize, searchEntity) {
        return $http.post('../order/findPage.do?pageNum=' + pageNum + "&pageSize=" + pageSize, searchEntity)
    };

    //删除方法
    this.removeOrderList = function (idList) {
        return $http.get('../order/removeOrder.do?idList=' + idList)
    };

    //根据id查询
    this.findOne = function (id) {
        return $http.get('../order/findOne.do?id=' + id)
    };

    //修改品牌
    this.updateOrder = function (order) {
        return $http.post('../order/updateOrder.do', order)
    };

    //新增品牌
    this.saveOrder = function (order) {
        return $http.post('../order/saveOrder.do', order)
    };   
});
